aee.js

```diff
@@ -82,3 +82,13 @@ Array.prototype.count = function (spec) {
   }
   return result
 }
+
+Array.prototype.sum = function (spec) {
+  const len = this.length
+  let sumSpec = (typeof spec === 'function') ? spec : spec => spec
```

Comment:
`... : spec => spec` is confusing, use a different name for the argument of arrow function rather than `spec`.